pkg/toolset/p2p_identity_extract.go

```diff
@@ -0,0 +1,81 @@
+package toolset
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	badger "github.com/ipfs/go-ds-badger"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-peerstore/pstoreds"
+)
+
+func extractP2PIdentity(args []string) error {
+	printUsage := func() {
+		println("Usage:")
+		println(fmt.Sprintf("	%s [P2P_STORE_PATH]", ToolP2PExtractIdentity))
+		println()
+		println("	[P2P_STORE_PATH]	- the path to the p2p store")
+		println()
+		println(fmt.Sprintf("example: %s %s", ToolP2PExtractIdentity, "./p2pstore"))
+	}
+
+	if len(args) != 1 {
+		printUsage()
+		return fmt.Errorf("wrong argument count '%s'", ToolSnapInfo)
```

Comment:
Thats the wrong tool. :)
```suggestion
		return fmt.Errorf("wrong argument count '%s'", ToolP2PExtractIdentity)
```